backend/qemu.pm

```diff
@@ -898,7 +906,7 @@ sub start_qemu ($self) {
             sp(lc($attribute), $vars->{$attribute}) if $vars->{$attribute};
         }
 
-        unless ($vars->{QEMU_NO_TABLET}) {
+        unless ($is_s390x or $vars->{QEMU_NO_TABLET}) {
```

Comment:
For conditions we usually use `||` (as it normally has the desired operator precedence for conditions):
```suggestion
        unless ($is_s390x || $vars->{QEMU_NO_TABLET}) {
```